components/profile-controller/ci/pipeline-run.yaml

```diff
@@ -15,4 +15,4 @@ spec:
   - name: $(image_name)
```

Comment:
How does $(generatName) work is this a feature in kustomize? 

I thought kustomize had the ability to generate unique suffixes but I thought that affected all resources.

In this case I think we only want to affect PipelineRun.

Reply:
> Should pipeline.yaml be embedded here using pipelineSpec (instead of pipelineRef)?

yes - see this example https://github.com/kubeflow/testing/issues/544#issuecomment-565694843

> How does $(generatName) work is this a feature in kustomize?

$(generateName) is a feature of kfctl so we can't go that route

> I thought kustomize had the ability to generate unique suffixes but I thought that affected all resources.

it's only for generators (secrets, configmaps) I believe. The prefix will affect all resources